tsconfig.bindings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "esModuleInterop": true,
+    "lib": ["dom", "es2015"],
+    "module": "es2015",
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "generated/bindings",
+    "removeComments": false,
+    "sourceMap": true,
+    "jsx": "react",
+    "target": "es2015"
+  },
+  "include": [
+    "generated/react-bindings/**/*.ts",
+    "generated/react-bindings/**/*.tsx"
+  ],
```

Comment:
This entire file was copied from a template, and the only changes are removing the `exclude`, changing the directories in `include`, and changing the `outDir`.

https://github.com/ionic-team/stencil-ds-plugins-demo/blob/04f0108046c02ca494dd6551b1dd822d16b87b8d/packages/component-library-react/tsconfig.json#L22-L25